web-basics-browser-security-model/activities/cockroach/sol/solution

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+PORT=8080
+
+if [[ $1 == "local" ]]
+then
+    remote='http://127.0.0.1:'$PORT
```

Comment:
If you update it, be kind enough to do it for all challenges

Reply:
Done.